src/partials/50_section-products.html

```diff
@@ -4,282 +4,897 @@
       <div class="row no-gutters align-items-end">
         <div class="col-auto heading"><h3>New furniture</h3></div>
         <div class="col menu">
-          <ul>
-            <li><a href="#" class="active">Bed</a></li>
-            <li><a href="#">Chair</a></li>
-            <li><a href="#">Sofa</a></li>
-            <li><a href="#">Table</a></li>
-            <li><a href="#">Dining</a></li>
-          </ul>
-        </div>
-        <div class="col-auto dots">
-          <ul>
-            <li><a href="#" class="active"></a></li>
-            <li><a href="#"></a></li>
-            <li><a href="#"></a></li>
+          <ul class="nav" id="categoriesTab" role="tablist">
+            <li class="nav-item">
+              <a
+                class="chair active"
+                id="chair-tab"
+                data-toggle="tab"
+                href="#chair"
+                role="tab"
+                aria-controls="chair"
+                aria-selected="true"
+                >Chair</a
+              >
+            </li>
+            <li class="nav-item">
+              <a
+                class="bed"
+                id="bed-tab"
+                data-toggle="tab"
+                href="#bed"
+                role="tab"
+                aria-controls="bed"
+                aria-selected="false"
+                >Bed</a
+              >
+            </li>
+            <li class="nav-item">
+              <a
+                class="sofa"
```

Comment:
W trybie mobile po kliknięciu w kategorię sofa na kropkę w prawym górnym rogu i następnie przesunięciu slajdera zaznaczona kropka zostaje aktywna + zaznacza się kropka w aktywnym slajdzie. Próbowałem znaleźć rozwiązanie ale chyba trzeba w ustawieniach tiny slider coś pozmieniać :)
![review](https://user-images.githubusercontent.com/49732790/63253182-a3fef980-c271-11e9-80f9-54d8f895e838.jpg)

Reply:
Ja tego nie widzę :( u mnie działa prawidłowo - zobacz:
https://drive.google.com/file/d/1UKzmHr91HfQzAk2EonBJr9uteBNf0RAP/view?usp=sharing

Reply:
Dlatego nie dawałem request changes bo nie byłem pewny czy to właśnie coś po mojej stronie nie tak.